web-app-router/scenes/Portal/Teams/TeamId/Apps/AppId/Actions/page/index.tsx

```diff
@@ -1,38 +1,47 @@
 import { DecoratedButton } from "@/components/DecoratedButton";
-import { IncognitoActionIcon } from "@/components/Icons/IncognitoActionIcon";
 import { LogoLinesIcon } from "@/components/Icons/LogoLines";
 import { WorldcoinBlueprintIcon } from "@/components/Icons/WorldcoinBlueprintIcon";
-import { ActionsList } from "./ActionsList/ActionsList";
 import { CreateActionModal } from "./CreateActionModal";
+import { getSdk as GetActionsSdk } from "../graphql/server/actions.generated";
+import { ActionsList } from "./ActionsList";
+import { getAPIServiceGraphqlClient } from "@/legacy/backend/graphql";
```

Comment:
@andy-t-wang Let's not import from legacy. I've created legacy folder temporarily to remove it after. Duplicate this in `@/lib`